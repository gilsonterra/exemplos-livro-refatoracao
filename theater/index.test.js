const index = require('./index');

test('test statement theater', () => {
    const invoice = require('./invoices.json');
    const plays = require('./plays.json');    

    expect(index.statement(invoice, plays)).toMatchSnapshot();
});