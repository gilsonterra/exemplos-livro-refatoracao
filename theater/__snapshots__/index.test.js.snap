// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test statement theater 1`] = `
"Statement for BigCo
Hamlet: $650.00 (55 seats)
As you like it: $580.00 (35 seats)
Othello: $500.00 (40 seats)
Amount owed is $1,730.00
You earned 47 credits.
"
`;
